test/validation/api/thread/thread.c

```diff
@@ -81,20 +82,29 @@ static int thread_global_term(odp_instance_t inst)
 
 static void thread_test_odp_cpu_id(void)
 {
-	(void)odp_cpu_id();
-	CU_PASS();
+	CU_ASSERT(odp_cpu_id() >= 0);
 }
 
 static void thread_test_odp_thread_id(void)
 {
-	(void)odp_thread_id();
-	CU_PASS();
+	int id = odp_thread_id();
+
+	CU_ASSERT(id >= 0);
+	CU_ASSERT(id < odp_thread_count_max());
+	CU_ASSERT(id < ODP_THREAD_COUNT_MAX);
 }
 
 static void thread_test_odp_thread_count(void)
 {
-	(void)odp_thread_count();
-	CU_PASS();
+	int count = odp_thread_count();
+
+	/* One thread running */
+	CU_ASSERT(count == 1);
```

Comment:
Leftover?

Reply:
Nope, there is only one thread running, so the return value should be one. The checks below are overlapping but give extra information in case of an invalid count is returned.

Reply:
OK.